eng/testing/linker/trimmingTests.targets

```diff
@@ -0,0 +1,132 @@
+<Project>
+  <Import Project="$(RepoRoot)eng\targets\ResolveReferences.targets" />
```

Comment:
Ensure this is still needed. I think it is needed for `CodeGen.proj` to work correctly.